tests/ast/test_basic.py

```diff
@@ -96,6 +96,23 @@ def test_get_user_nodes_excluded():
     plus_assign = a_var.get_user_nodes(Assign, excluded_nodes = PyccelMinus)
     assert(len(plus_assign)==2)
 
+def test_get_all_user_nodes():
+    filename = os.path.join(path_dir, "math.py")
+
+    interesting_var = Variable('int', 'b')
+
+    fst = get_functions(filename)[0]
+    atts = set(fst.get_attribute_nodes(Variable))
+    atts = [v for v in atts  if v == interesting_var]
+
+    a_var = atts[0]
+
+    users = set(a_var.get_all_user_nodes())
+
+    assert(all(isinstance(s, (PyccelMul, PyccelMinus, AugAssign, FunctionDefArgument)) for s in users))
+
+    assert(len(users) == 4)
```

Comment:
Is there any hope that we could remove the parentheses from the `assert` statements in this file before it gets too long?

Reply:
Done 4676de74

Reply:
Thanks! 😃